dot/state/storage.go

```diff
@@ -30,7 +30,7 @@ func errTrieDoesNotExist(hash common.Hash) error {
 // StorageState is the struct that holds the trie, db and lock
 type StorageState struct {
 	blockState *BlockState
-	tries      *sync.Map // map[common.Hash]*trie.Trie // map of root -> trie
+	rootToTrie *rootToTrieMap
```

Comment:
```suggestion
	tries *rootToTrieMap
```
nit: can we keep the same syntax.  I kinda like `s.tries`.

Reply:
done in d7b3f3ffe256a3aa9fa602c5ddd35a414fe177a8